.github/workflows/MigrationCodelab.yaml

```diff
@@ -26,10 +26,10 @@ jobs:
       - name: Copy CI gradle.properties
         run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties        
 
-      - name: Set up JDK 1.8
+      - name: Set up JDK 11
         uses: actions/setup-java@v1
         with:
-          java-version: 1.8
+          java-version: 11
```

Comment:
Is this something we should do in each codelab & sample? Maybe it warrants going through them systematically after the alpha10 update?

Reply:
AGP 7.0.0 requires Java 11 now. As this codelab couldn't be updated in previous releases, I'm adding the change here. The rest of codelabs are already using Java 11